src/openrct2-ui/interface/Widget.cpp

```diff
@@ -1067,14 +1076,15 @@ static void widget_text_box_draw(rct_drawpixelinfo* dpi, rct_window* w, rct_widg
 
     gCurrentFontSpriteBase = FONT_SPRITE_BASE_MEDIUM;
     gCurrentFontFlags = 0;
+    ScreenCoordsXY screenCoords(l + 2, t);
```

Comment:
```suggestion
    const ScreenCoordsXY screenCoords(l + 2, t);
```
Just to make sure people know it's not meant to be altered (as it affects two places)